Optionally export the AWS_SESSION_TOKEN

blacs30 opened the pull request:
When using onelogin-aws-cli the AWS_SESSION_TOKEN is written into the ~/.aws/credentials file for a profile.
That token has to be exported as well to be able to use e.g. the aws-cli.

This PR adds the function to also export the AWS_SESSION_TOKEN if present.

blacs30 committed:
Export also the aws_session_token if it exist in the credentials file,
that is the case i.e. when using the onelogin-aws-cli.
Update tests.py and increment version to 1.2.0

modified:   setup.py
modified:   src/awsenv/__init__.py
modified:   src/awsenv/tests.py

naftulikay commented on src/awsenv/tests.py:
```diff
@@ -20,7 +20,7 @@ def test_load(self):
 
         self.assertIsNotNone(detected_path)
-        self.assertIn(detected_path, ['/usr/bin/gpg2', '/usr/bin/gpg'])
+        self.assertIn(detected_path, ['/usr/bin/gpg2', '/usr/bin/gpg', '/usr/local/MacGPG2/bin/gpg2'])
```
this needs to be removed

blacs30 replied:
I've removed the hard coded path.

naftulikay reviewed (changes requested):
Excellent work, thank you for your contribution! I think if you remove the reference to your hardcoded GnuPG binary path and squash, should be ready for merge.

naftulikay reviewed (approved):
LGTM :+1: Please squash and I'll merge.

blacs30 commented:
@naftulikay I've squashed my commits. Hope it looks good (honestly it was the first time I squashed something.. good to have learned little bit about that.)

naftulikay commented:
Merged, thank you so much for your contribution!

naftulikay commented:
Released in [v1.2.0](https://github.com/naftulikay/aws-env/releases/tag/v1.2.0).
